packs/journals/archetypes.json

```diff
@@ -3546,6 +3546,27 @@
                 "controls": true,
                 "volume": 0.5
             }
+        },
+        {
+            "_id": "DOc3Pf8wmVxanTIv",
+            "image": {},
+            "name": "Spellcasting Archetypes",
+            "sort": 100000,
+            "src": null,
+            "system": {},
+            "text": {
+                "content": "<p>Some archetypes grant you a substantial degree of spellcasting, albeit delayed compared to a character from a spellcasting class. A spellcasting archetype allows you to use scrolls, staves, and wands in the same way that a member of a spellcasting class can.</p><p>Spellcasting archetypes always grant the ability to cast cantrips in their dedication, and then they have a basic spellcasting feat, an expert spellcasting feat, and a master spellcasting feat. These feats share their name with the archetype; for instance, the druid’s master spellcasting feat is called Master Druid Spellcasting. All spell slots you gain from spellcasting archetypes are subject to the restrictions within the archetype. For instance, the witch archetype allows you to pick a patron when you take its dedication feat. If you pick patron granting occult spells, the archetype then grants you spell slots you can use only to cast occult spells you prepare as a witch, even if you are a bard with occult spells in your repertoire.</p><h2>Basic Spellcasting Feat</h2><p>Usually available at 4th level, these feats grant a 1st-rank spell slot. At 6th level, they grant you a 2nd-rank spell slot, and if you have a spell repertoire, you can select one spell from your repertoire as a signature spell. At 8th level, they grant you a 3rd-rank spell slot. Archetypes refer to these benefits as the “basic spellcasting benefits.”</p><h2>Expert Spellcasting Feat</h2><p>Typically taken at 12th level, these feats make you an expert in spell attack rolls and DCs of the appropriate magical tradition and grant you a 4thrank spell slot. If you have a spell repertoire, you can select a second spell from your repertoire as a signature spell. At 14th level, they grant you a 5th-rank spell slot, and at 16th level, they grant you a 6th-rank spell slot. Archetypes refer to these benefits as the “expert spellcasting benefits.”</p><h2>Master Spellcasting Feat</h2><p>Usually found at 18th level, these feats make you a master in spell attack rolls and DCs of the appropriate magical tradition and grant you a 7thrank spell slot. If you have a spell repertoire, you can select a third spell from your repertoire as a signature spell. At 20th level, they grant you an 8th-rank spell slot. Archetypes refer to these benefits as the “master spellcasting benefits.”</p>",
```

Comment:
```suggestion
                "content": "<p>Some archetypes grant you a substantial degree of spellcasting, albeit delayed compared to a character from a spellcasting class. A spellcasting archetype allows you to use scrolls, staves, and wands in the same way that a member of a spellcasting class can.</p><p>Spellcasting archetypes always grant the ability to cast cantrips in their dedication, and then they have a basic spellcasting feat, an expert spellcasting feat, and a master spellcasting feat. These feats share their name with the archetype; for instance, the druid's master spellcasting feat is called Master Druid Spellcasting. All spell slots you gain from spellcasting archetypes are subject to the restrictions within the archetype. For instance, the witch archetype allows you to pick a patron when you take its dedication feat. If you pick patron granting occult spells, the archetype then grants you spell slots you can use only to cast occult spells you prepare as a witch, even if you are a bard with occult spells in your repertoire.</p><h2>Basic Spellcasting Feat</h2><p>Usually available at 4th level, these feats grant a 1st-rank spell slot. At 6th level, they grant you a 2nd-rank spell slot, and if you have a spell repertoire, you can select one spell from your repertoire as a signature spell. At 8th level, they grant you a 3rd-rank spell slot. Archetypes refer to these benefits as the \"basic spellcasting benefits.\"</p><h2>Expert Spellcasting Feat</h2><p>Typically taken at 12th level, these feats make you an expert in spell attack rolls and DCs of the appropriate magical tradition and grant you a 4thrank spell slot. If you have a spell repertoire, you can select a second spell from your repertoire as a signature spell. At 14th level, they grant you a 5th-rank spell slot, and at 16th level, they grant you a 6th-rank spell slot. Archetypes refer to these benefits as the \"expert spellcasting benefits.\"</p><h2>Master Spellcasting Feat</h2><p>Usually found at 18th level, these feats make you a master in spell attack rolls and DCs of the appropriate magical tradition and grant you a 7thrank spell slot. If you have a spell repertoire, you can select a third spell from your repertoire as a signature spell. At 20th level, they grant you an 8th-rank spell slot. Archetypes refer to these benefits as the \"master spellcasting benefits.\"</p>",
```

Reply:
Well, it won't let me apply this for some reason, but get rid of the rsquo in "druid's" and replace the double quotes with standard `"` (escaping if editing the raw json).